deposit/src/test/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJobIT.java

```diff
@@ -728,6 +734,135 @@ public void onlyIngestionEventAddedTest() throws Exception {
+        depBag.add(folder3Bag);
+
+        job.closeModel();
+
+        job.run();
+
+        treeIndexer.indexAll(baseAddress);
+
+        // Verify that the correct original deposit ids are assigned to each folder
+        FolderObject folder1 = repoObjLoader.getFolderObject(folderObj1Pid);
+        Resource f1DepositResc = folder1.getResource().getProperty(Cdr.originalDeposit).getResource();
+        assertEquals(deposit2Pid.getRepositoryPath(), f1DepositResc.getURI());
+
+        FolderObject folder2 = repoObjLoader.getFolderObject(folderObj2Pid);
+        Resource f2DepositResc = folder2.getResource().getProperty(Cdr.originalDeposit).getResource();
+        assertEquals(depositPid.getRepositoryPath(), f2DepositResc.getURI());
+
+        FolderObject folder3 = repoObjLoader.getFolderObject(folderObj3Pid);
+        Resource f3DepositResc = folder3.getResource().getProperty(Cdr.originalDeposit).getResource();
+        assertEquals(deposit3Pid.getRepositoryPath(), f3DepositResc.getURI());
+    }
+
+    private final static String CREATED_STRING = "2011-10-04T20:36:44.902Z";
+    private final static String LAST_MODIFIED_STRING = "2013-10-06T10:16:44.111Z";
+    private final static Date CREATED_DATE = DateTimeUtil.parseUTCToDate(CREATED_STRING);
+    private final static Date LAST_MODIFIED_DATE = DateTimeUtil.parseUTCToDate(LAST_MODIFIED_STRING);
+
+    @Test
+    public void overrideTimestampsTest() throws Exception {
+        Map<String, String> status = new HashMap<>();
+        status.put(DepositField.containerId.name(), RepositoryPaths.getContentRootPid().getRepositoryPath());
+        status.put(DepositField.permissionGroups.name(), "adminGroup");
+        status.put(DepositField.overrideTimestamps.name(), "true");
+        depositStatusFactory.save(depositUUID, status);
+
+        Model model = job.getWritableModel();
+        Bag depBag = model.createBag(depositPid.getRepositoryPath());
+
+        PID unitPid = pidMinter.mintContentPid();
+        Bag unitBag = model.createBag(unitPid.getRepositoryPath());
+        unitBag.addProperty(RDF.type, Cdr.AdminUnit);
+        unitBag.addLiteral(CdrDeposit.lastModifiedTime, LAST_MODIFIED_STRING);
+        unitBag.addLiteral(CdrDeposit.createTime, CREATED_STRING);
+        depBag.add(unitBag);
+
+        PID collPid = pidMinter.mintContentPid();
+        Bag collBag = model.createBag(collPid.getRepositoryPath());
+        collBag.addProperty(RDF.type, Cdr.Collection);
+        collBag.addLiteral(CdrDeposit.lastModifiedTime, LAST_MODIFIED_STRING);
+        collBag.addLiteral(CdrDeposit.createTime, CREATED_STRING);
+        unitBag.add(collBag);
+
+        PID folderPid = pidMinter.mintContentPid();
+        Bag folderBag = model.createBag(folderPid.getRepositoryPath());
+        folderBag.addProperty(RDF.type, Cdr.Folder);
+        folderBag.addLiteral(CdrDeposit.lastModifiedTime, LAST_MODIFIED_STRING);
```

Comment:
does .addLiteral() overwrite the previous value, or is the last modified on the deposit not set by default?

Reply:
it doesn't overwrite the previous value, unless its exactly the same value. The last modified is not normally set in the deposit pipeline because it is automatically generated by fedora. Fedora actually would normally throw an error if you make a PUT or a POST request with the created or lastModified timestamp, you have to put it into the special "relaxed" mode to allow this, which we will only have enabled when we need it (such as the migration). You can see me turning it on for this IT in this PR.

Reply:
ah, so it's something you generally don't need to worry about.